pkg/scheduler/apis/config/scheme/scheme.go

```diff
@@ -23,6 +23,7 @@ import (
 	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
```

Comment:
let's remove `kubescheduler` prefix from all these aliases

Reply:
Please do this in a followup

Reply:
Done.